lib/Doctrine/DBAL/Types/DateIntervalType.php

```diff
@@ -56,9 +58,13 @@ public function convertToPHPValue($value, AbstractPlatform $platform)
         }
 
         try {
-            return new \DateInterval($value);
+            $interval = new \DateInterval(substr($value, 1));
```

Comment:
With this you assume that `$value` always have the sign, which is not the case for existing data.
We should make sure that the type would still work for the "old" format.